hardware/esp8266com/esp8266/libraries/ESP8266WebServer/src/ESP8266WebServer.h

```diff
@@ -83,6 +83,13 @@ class ESP8266WebServer
   String argName(int i);          // get request argument name by number
   int args();                     // get arguments count
   bool hasArg(const char* name);  // check if argument exists
+  
+  void collectHeaders(const char* headerKeys[], const size_t headerKeysCount); // set the request headers to collect
+  String header(const char* name);   // get request header value by name
```

Comment:
If you look just above (ESP8266WebServer.h Line 81), you'll see that the existing argument access method is the same. There may be a better way but I decided to follow the conventions of the existing code.